File: app/src/main/java/com/example/aamezencev/weatherinfo/data/owmApi/JsonCoordInfo.java
package com.example.aamezencev.weatherinfo.data.owmApi;

import com.google.gson.annotations.SerializedName;

/**
 * Created by aa.mezencev on 18.01.2018.
 */

public class JsonCoordInfo {
    @SerializedName("lon")
    private String lon;
    @SerializedName("lat")
    private String lat;

    public JsonCoordInfo(String lon, String lat) {
        this.lon = lon;
        this.lat = lat;
    }

    public JsonCoordInfo() {

    }

    public String getLat() {
        return lat;
    }

    public String getLon() {
        return lon;
    }
}

File: app/src/main/java/com/example/aamezencev/weatherinfo/data/owmApi/JsonCoordInfo.kt
package com.example.aamezencev.weatherinfo.data.owmApi

import com.google.gson.annotations.SerializedName

/**
 * Created by aa.mezencev on 18.01.2018.
 */

data class JsonCoordInfo(@field:SerializedName("lon") val lon: String? = "",
                    @field:SerializedName("lat") val lat: String? = "") {

}
